app/home/-components/testimonials/styles.scss

```diff
@@ -1,187 +1,135 @@
+// stylelint-disable
 @import url('https://fonts.googleapis.com/css?family=Bree+Serif|Coustard');
 
-.container {
+.visually-hidden {
+    display: none;
+}
+
+.testimonials-container {
+    display: grid;
+    grid-template-rows: auto;
+    grid-template-columns: minmax(min-content, 1fr);
+    grid-template-areas:
+        "head"
+        "slideData";
     background-color: #ecf0f1;
 
     header {
-        padding: 32px 20px 10px;
-    }
-
-    h1 {
-        font-size: 48px;
-        font-weight: 600;
+        grid-area: head;
+        justify-content: center;
         text-align: center;
-        max-width: 785px;
-        margin: 0 auto;
-        color: #263947;
+        background-color: #ecf0f1;
+        padding: 10px 20px 0px;
+
+        h1 {
+            text-align: center;
+            font-family: 'Open Sans', 'Helvetica Neue', sans-serif;
+            font-size: 5.0vmin;
+            font-weight: 600;
+            color: #263947;
+            margin-bottom: 3px;
+            margin-top: 27px;
+        }
     }
 }
 
 .slide-content {
-    display: flex;
-    height: inherit;
-    padding: 10px 50px 0;
-    justify-content: space-evenly;
-    max-width: 1200px;
-    margin: 0 auto;
+    grid-area: slideData;
+    display: inline-flex;
 }
 
-.slide-image {
-    max-height: 100%;
-    align-self: flex-end;
+.slide-content > li {
+    grid-template-areas:
+        "researchLeft researcherProfile";
+    justify-content: flex-end;
+    align-items: flex-end;
+    bottom: 0;
+    width: 100%;
+    object-fit: cover;
+    max-height: 420px;
 }
 
-.slide-content {
-    height: 420px;
-
-    p {
-        max-width: 444px;
-        font-family: Bree Serif;
-        font-size: 32px;
-        color: #365063;
-        text-decoration: none solid rgb(54, 80, 99);
-        line-height: 1.5em;
-        text-align: center;
-    }
+.testimonials-left {
+    grid-area: researchLeft;
+    margin: 0;
+    padding: 0;
+}
 
-    .testimonial-small {
-        font-size: 20px;
-    }
+.testimonials-left > span {
+    text-align: center;
 
-    .description {
-        font-family: Open Sans;
-        height: 66px;
+    p,
+    a {
+        font-family: 'Open Sans';
         font-size: 16px;
         text-align: center;
+        line-height: 0.7vh;
+        padding: 6px 0px 9px;
+        margin: 3px 0px;
     }
 }
 
-.quote {
-    height: 48px;
-    display: block;
-    padding: 20px;
+.single-quote {
     font-family: Coustard;
-    font-size: 60px;
     color: #a5b3bd;
+    justify-content: center;
+    font-size: 6.0vmin;
+    line-height: 40px;
+    text-decoration: none solid #a5b3bd;
+    margin: -11px 0px 0px;
+    top: -31px;
+    width: 100%;
+}
+
+.researcher-quote {
+    grid-area: researcherQuote;
+    font-family: 'Bree Serif';
```

Comment:
You should generally have a fallback generic font-family in case the user doesn't have Bree Serif. For example, `font-family: 'Bree Serif' , serif;`